src/components/Dropdown.js

```diff
@@ -94,8 +93,8 @@ export default class Dropdown extends React.Component {
     if (this.refs.list) this.refs.list.handleKeys(event)
   }
 
-  chooseChild = choice => {
-    const link = findChildLink(findDOMNode(this.refs[choice.ref]))
+  chooseChild = (element, node) => {
+    const link = findChildLink(node)
```

Comment:
oh dang. that's kind of a hidden constraint on contents of KeyControlledList. if there isn't a way to work around it, let's at least make a note of this in the comments on the component.